zc_common/remote_resource/request.py

```diff
@@ -25,14 +25,7 @@ class RouteNotFoundException(Exception):
 
 
 class ServiceRequestException(Exception):
-    """An exception commonly thrown when an HTTP request to another service endpoint fails."""
-    message = None
-    response = None
-
-    def __init__(self, message, response):
```

Comment:
What if we made these kwargs instead, that way we don't necessarily lose the information presented. That said, I don't know how useful the information in the response is.

```
def __init__(self, *args, **kwargs):
    self.message = kwargs.pop('message', '')
    self.response = kwargs.pop('response', '')
    super(ServiceRequestException, self).__init__(self.message)
```

Wouldn't break the init of exception?